src/components/flow/flow.stories.ts

```diff
@@ -38,8 +38,8 @@ const createFlowItemAttributes: (group: string) => Attributes = (group) => {
       value: boolean("menuOpen", false, group)
     },
     {
-      name: "summary",
-      value: text("summary", "Summary", group)
+      name: "description",
+      value: text("description", "Description", group)
```

Comment:
@anveshmekala I updated the flow story to set the panel to use `description` instead of summary. Any other stories we should check to make sure they aren't using the deprecated props?

Reply:
This should be in a separate PR as it's unrelated to the overall changes.

Reply:
Ok. I'll revert.